core/include/detray/tools/bin_fillers.hpp

```diff
@@ -0,0 +1,139 @@
+/** Detray library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
```

Comment:
2022 -> 2023, you might also want to change it in `CMakeLists.txt` and `grid_builder.hpp`

Reply:
OK